goreleaser/artifacts.json

```diff
@@ -0,0 +1 @@
+[{"name":"metadata.json","path":"goreleaser/metadata.json","internal_type":30,"type":"Metadata"},{"name":"pulumi-converter-terraform","path":"goreleaser/pulumi-converter-terraform_linux_amd64_v1/pulumi-converter-terraform","goos":"linux","goarch":"amd64","goamd64":"v1","internal_type":4,"type":"Binary","extra":{"Binary":"pulumi-converter-terraform","Ext":"","ID":"pulumi-converter-terraform"}}]
```

Comment:
I assume these goreleaser bits (especially the binary below) are not supposed to go in here?

Reply:
No they weren't meant to be, though I wonder if we can automate releases more easily like providers do.